utils/logger.ts

```diff
@@ -24,6 +24,7 @@ export async function setLoggerUser(vue: Vue, { wallet } : { wallet?: string } =
       // https://support.google.com/analytics/answer/9973999?hl=en
       // vue.$gtag.config({ user_id: hashedId });
       vue.$gtag.set({ user_id: hashedId });
+      vue.$gtag.event('login', { method });
```

Comment:
Do we know if it's explicit login or just restore session? We don't want to call login event if it's just restored session